rllib/core/learner/learner.py

```diff
@@ -272,6 +281,40 @@ def __init__(
         # the final results dict in the `self.compile_update_results()` method.
         self._metrics = defaultdict(dict)
 
+    @OverrideToImplementCustomLogic_CallToSuperRecommended
+    def build(self) -> None:
+        """Builds the Learner.
+
+        This method should be called before the learner is used. It is responsible for
+        setting up the RLModule, optimizers, and (optionally) their lr-schedulers.
+        """
+        if self._is_built:
+            logger.debug("Learner already built. Skipping build.")
+            return
+
+        # Build learner connector pipeline used on this Learner worker.
+        # TODO (sven): Support multi-agent cases.
+        if self.config.uses_new_env_runners and not self.config.is_multi_agent():
```

Comment:
For now: Only on new API stack + EnvRunner + single-agent: use Learner connector (w/o this PPO on new stack would not learn).